Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py

```diff
@@ -74,7 +74,7 @@ def http_request(self, method, url_suffix, full_url=None, headers=None, json_dat
                                     f'\n{res}')
 
         except Exception as e:
-            if 'SSL Certificate Verification Failed' in e.args[0]:
+            if self._verify and 'SSL Certificate Verification Failed' in e.args[0]:
```

Comment:
shouldnt this be `if not self._verify` ?

Reply:
no, if the verify is true, it means that the checkbox is disabled (insecure is "not verify").